### DISCUSSION TITLE
How to properly setup Build phase settings for two different apps?

### DISCUSSION THREAD

**Discussion by drinikol** [invertase/react-native-firebase#4234] [Q&A] [answered]
We have an app which has two Google Service info.plist and two Info.plist, how to properly set it up? Our workspace project has two apps One for Student, one for teacher. I am having a hard time figuring it out. Hopefully some one may enlighten me about this. Thank you! I would really appreciate the help.

**Comment by andersonaddo** [comment, marked as answer]
https://rnfirebase.io/faqs-and-tips#tips

One bullet point addresses this
